src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArworkSearch.tsx

```diff
@@ -0,0 +1,123 @@
+import { Box, Button, Clickable, Spacer, Text } from "@artsy/palette"
+import ArtworkGridItemFragmentContainer from "Components/Artwork/GridItem"
+import { Masonry } from "Components/Masonry"
+import { Fragment } from "react"
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { extractNodes } from "Utils/extractNodes"
+import { MyCollectionArworkSearchQuery } from "__generated__/MyCollectionArworkSearchQuery.graphql"
+
+interface MyCollectionArworkSearchProps {
+  artistId: string
+  onClick: (artwork: any) => void
+  onSkip: () => void
+  query?: string | null
+}
+
+export const MyCollectionArworkSearch: React.FC<MyCollectionArworkSearchProps> = ({
+  artistId,
+  onClick,
+  onSkip,
+  query,
+}) => {
+  const input = query ? { keyword: query } : null
+
+  const data = useLazyLoadQuery<MyCollectionArworkSearchQuery>(
+    graphql`
+      query MyCollectionArworkSearchQuery(
+        $artistID: String!
+        $input: FilterArtworksInput
+      ) {
+        artist(id: $artistID) {
+          filterArtworksConnection(first: 30, input: $input) {
+            edges {
+              node {
+                medium
+                date
+                depth
+                editionSize
+                editionNumber
+                height
+                images {
+                  height
+                  isDefault
+                  imageURL
+                  width
+                }
+                id
+                internalID
+                isEdition
+                category
+                metric
+                title
+                width
+                ...GridItem_artwork
+              }
+            }
+          }
+        }
+      }
+    `,
+    { artistID: artistId, input }
+  )
+
+  const artworks = extractNodes(data.artist?.filterArtworksConnection)
+
+  if (artworks.length === 0) {
+    return <NoResults onSkip={onSkip} query={query} />
+  }
+
+  return (
+    <>
+      <Text variant={["xs", "sm-display"]}>
+        Or skip ahead to{" "}
+        <Clickable onClick={onSkip} textDecoration="underline">
+          <Text>add artwork details.</Text>
```

Comment:
Let's move the dot out of the clickable so it's not underlined?

Reply:
Good idea 👍